portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -37,14 +37,18 @@
   private static final String REFLECTION_PARAMETER = "reflection";
   private static final String COMMENT_PARAMETER = "Comment";
   private static final String TIMESTAMP_PARAMETER = "timestamp";
+  private static final String MAXCOMMENT_PARAMETER = "max-comment";
   private static final String APPLICATION_JSON_PARAMETER = "application/json;";
   private static final String COMMENT_HTML_PARAMETER = "/comments.html";
 
+  private static final int MAXCOMMENT = 5;//hardcoded soon to be taken in by user input
 
   private String userNameInput;
   private String reflectionInput;
   private long idGiven;
-  private long timestampOfComment;  
+  private long timestampOfComment; 
```

Comment:
Instead of storing `timestamp` as a `long` I would store it as a [`java.time.Instant`](https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html)